src/main.ts

```diff
@@ -17,7 +17,6 @@ process.on('unhandledRejection', (error) => {
   console.error(error);
```

Comment:
> Unexpected console statement

Reply:
Removed in the addTypes pr that I will open in a sec